### DISCUSSION TITLE
how to go back to the old icons?

### DISCUSSION THREAD

**Discussion by jabreen** [SpotX-Official/SpotX#20] [Ideas]
just wondering if it's possible, don't really like the new ones. thanks!

**Comment by amd64fox** [comment]
Hello, you can return old icons like this:
- Open folder `AppData\Roaming\Spotify\Apps`
- Open the `xpui.spa` file, for example using the `7zip archiver`
- Open the `xpui.js` file for editing and search for the line `Enables the 2021 icons redraw which loads a different font asset for rendering icon glyphs.",default:!0`
- Change at the end of the found text this `default:!0` to this `default:!1`
- save file and close

Leave your comment here if someone doesn't like the new icons either.

**Reply by jabreen** [reply]
thank you very much!

**Comment by amd64fox** [comment]
starting from version 1.1.84.716, it is no longer possible to change icons, now all users have only new icons.
